app/views/projects/samples/index.html.erb

```diff
@@ -31,18 +31,90 @@
     </div>
   <% end %>
 <% end %>
-
-<div class="bg-white dark:bg-slate-800">
-  <div class="flex flex-col">
-    <div class="overflow-x-auto">
-      <div class="shadow">
-        <% if @has_samples %>
+<% if @has_samples %>
+  <div
+      class="
+        flex
+        text-sm
+        font-medium
+        text-center
+        border-b
+        text-slate-500
+        border-slate-200
+        dark:text-slate-400
+        dark:border-slate-700
+      "
+    >
+    <div class="flex flex-row items-center ml-auto space-x-2 font-normal mb-4">
```

Comment:
Nitpick: Can you update `mb-4` to `mb-2` so the search and sort sit closer to the table like on the project dashboard page.

Reply:
Fixed in 8300c9fa3d318925f5cc6eb73f67203af6d9d53c